packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -1168,3 +1179,36 @@ function viewParticipants() {
   }
 }
 
+/* ANSWER/REJECT INCOMING CALL */
+
+function toggleDisplay(elementId, status) {
+  const element = document.getElementById(elementId);
+
+  if (status) {
+    element.style.visibility = 'visible';
+  }
+  else {
+    element.style.visibility = 'hidden';
+  }
+}
+
+function answerMeeting() {
+  const meeting = getCurrentMeeting();
+
+  if (meeting) {
+    meeting.join().then(() => {
+      meeting.acknowledge('ANSWER', false);
+    });
+    toggleDisplay('incomingsection', false);
+  }
+}
+
+function rejectMeeting() {
```

Comment:
Good find! Glad you were able to implement these blocks [i must have overlooked this during design].